ansible/roles/bigbluebutton/molecule/default/molecule.yml

```diff
@@ -27,7 +28,6 @@ provisioner:
     links:
       group_vars: ../../../../inventory/molecule/group_vars/
   playbooks:
-    prepare: prepare.yml
```

Comment:
The `prepare.yml` is still in the commit. So we either remove it or we don't remove the above line. But not both.

Reply:
I missed that!
The playbook in `prepare.yml` is required, as you can see in the the CI log it is running: https://travis-ci.org/github/PrivacyLx/devops/jobs/763911247#L594

I have added it again in: https://github.com/PrivacyLx/devops/pull/55/commits/5614b609c20a3a38a78098f994ac46f2f7a44840

Reply:
perfect.